config/initializers/countries.rb

```diff
@@ -0,0 +1,16 @@
+country_config = {
+  'IN': {
+    abbreviation: 'IN',
+    name: 'India',
+    dashboard_locale: ENV['DEFAULT_PREFERRED_DASHBOARD_LOCALE'] || 'en_IN',
+    time_zone: ENV['DEFAULT_TIME_ZONE'] || 'Asia/Kolkata'
+  },
+  'BD': {
+    abbreviation: 'BD',
+    name: 'Bangladesh',
+    dashboard_locale: ENV['DEFAULT_PREFERRED_DASHBOARD_LOCALE'] || 'en_BD',
+    time_zone: ENV['DEFAULT_TIME_ZONE'] || 'Asia/Dhaka'
+  }
+}.with_indifferent_access
+
+Rails.application.config.country = country_config[ENV.fetch('DEFAULT_COUNTRY')]
```

Comment:
```suggestion
Rails.application.config.default_country = country_config[ENV.fetch('DEFAULT_COUNTRY')]
```

Reply:
Why do we use the `default` terminology? Can an instance of Simple switch between countries effectively? In other words, can a single instance of Simple show reports in IST _and_ also in Bangla time?